### PR TITLE
modify dir strucure and bugfix

### PR DESCRIPTION
**Pull request by zhcn381**

### CODE REVIEW

File: config [hunk 1]
```diff
@@ -76,54 +76,54 @@ HTTP_MODULES="$HTTP_MODULES ngx_http_tfs_module"
...
-		$ngx_addon_dir/ngx_http_tfs_remote_block_cache.h \
-		$ngx_addon_dir/ngx_http_tfs_timers.h \
-		$ngx_addon_dir/ngx_http_tfs_rc_server_message.h \
-		$ngx_addon_dir/ngx_http_tfs_name_server_message.h \
-		$ngx_addon_dir/ngx_http_tfs_data_server_message.h \
-		$ngx_addon_dir/ngx_http_tfs_root_server_message.h \
-		$ngx_addon_dir/ngx_http_tfs_meta_server_message.h \
-		$ngx_addon_dir/ngx_http_tfs_peer_connection.h \
-		$ngx_addon_dir/ngx_http_tfs_server_handler.h "
+                $ngx_addon_dir/src/ngx_http_tfs.h \
```
Comment (yaoweibin) [inline config]: 对齐

File: config [hunk 2]
```diff
@@ -76,54 +76,54 @@ HTTP_MODULES="$HTTP_MODULES ngx_http_tfs_module"
...
+		$ngx_addon_dir/src/ngx_http_tfs_errno.h \
+                $ngx_addon_dir/src/ngx_tfs_common.h \
```
Comment (yaoweibin) [inline config]: 对齐

File: config [hunk 3]
```diff
@@ -76,54 +76,54 @@ HTTP_MODULES="$HTTP_MODULES ngx_http_tfs_module"
...
-		$ngx_addon_dir/ngx_http_tfs_remote_block_cache.c \
-		$ngx_addon_dir/ngx_http_tfs_timers.c \
-		$ngx_addon_dir/ngx_http_tfs_rc_server_message.c \
-		$ngx_addon_dir/ngx_http_tfs_name_server_message.c \
-		$ngx_addon_dir/ngx_http_tfs_data_server_message.c \
-		$ngx_addon_dir/ngx_http_tfs_root_server_message.c \
-		$ngx_addon_dir/ngx_http_tfs_meta_server_message.c \
-		$ngx_addon_dir/ngx_http_tfs_peer_connection.c \
-		$ngx_addon_dir/ngx_http_tfs_server_handler.c "
+                $ngx_addon_dir/src/ngx_tfs_common.c \
```
Comment (yaoweibin) [inline config]: 同上

**Comment by monadbobo** [comment]
looks good to me!

**Pull request merged**